topics/sequence-analysis/tutorials/ncbi-blast-against-the-madland/tutorial.md

```diff
@@ -0,0 +1,172 @@
+>    - _"Protein query sequence(s)"_: `Amino acid input sequence` (In case of BLASTp) *OR* 
+>    - _"Translated nucleotide query sequence(s)"_: `Translated nucleotide input sequence` (In case of BLASTx) 
+>    - _"Subject database/sequences"_: `Locally installed BLAST database`
+>    - _"Protein BLAST database"_: `MadLandDB (Genome zoo) plant and algal genomes with a focus on non-seed plants and streptophyte algae (22 Dec 2022)`
+>    - _"Set expectation value cutoff"_: `0.001`
+>    - _"Output format"_:
+>    - In _"Output Options"_: `Tabular (extended 25 columns)` 
+> <img src="../../images/ncbi-blast-against-the-madland/blast-example.png" alt="blast against madland" width="80%">
+>
+{: .hands_on}
+
+## Blast output
+
+> {% icon tool %} The BLAST output will be in tabular format (you can select the desired output format from the drop down menu) and include the following fields : 
+
+>
+
+<table>
+  <tr>
+    <th>Column</th>
+    <th>NCBI name</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td>1</td>
+    <td>qseqid</td>
+    <td>Query Seq-id (ID of your sequence)</td>
+  </tr>
+  <tr>
+    <td>2</td>
+    <td>sseqid</td>
+    <td>Subject Seq-id (ID of the database hit)</td>
+  </tr>
+   <tr>
+    <td>3</td>
+    <td>pident</td>
+    <td>Percentage of identical matches</td>
+  </tr>
+   <tr>
+    <td>4</td>
+    <td>length</td>
+    <td>Alignment length</td>
+  </tr>
+   <tr>
+    <td>5</td>
+    <td>mismatch</td>
+    <td>Number of mismatches</td>
+  </tr>
+   <tr>
+    <td>6</td>
+    <td>gapopen</td>
+    <td>Number of gap openings</td>
+  </tr>
+   <tr>
+    <td>7</td>
+    <td>qstart</td>
+    <td>Start of alignment in query</td>
+  </tr>
+   <tr>
+    <td>8</td>
+    <td>qend</td>
+    <td>End of alignment in query</td>
+  </tr>
+   <tr>
+    <td>9</td>
+    <td>sstart</td>
+    <td>Start of alignment in subject (database hit)</td>
+  </tr>
+   <tr>
+    <td>10</td>
+    <td>send</td>
+    <td>End of alignment in subject (database hit)</td>
+  </tr>
+    <tr>
+    <td>11</td>
+    <td>evalue</td>
+    <td>Expectation value (E-value)</td>
+  </tr>
+    <tr>
+    <td>12</td>
+    <td>bitscore</td>
+    <td>Bit score</td>
+  </tr>
+</table>
+
+> The fields are separated by tabs, and each row represents a single hit. For more details for BLAST analysis and output, we recommand you to follow the [Similarity-searches-blast](https://training.galaxyproject.org/training-material/topics/genome-annotation/tutorials/genome-annotation/tutorial.html#similarity-searches-blast) tutorial.
+
+
+> <details-title>Further Reading about BLAST Tools in Galaxy</details-title>
+>
+>
+> Cock et al. (2015): [NCBI BLAST+ integrated into Galaxy](http://biorxiv.org/content/early/2015/05/04/014043.full-text.pdf+html)
+>
+> Cock et al. (2013): [Galaxy tools and workflows for sequence analysis with applications in molecular plant pathology](https://peerj.com/articles/167/)
+{: .details}
```

Comment:
If you could replace these with proper citations as well (see comment below) that'd be really helpful!

Reply:
Changes have been made, Thank you :)